app/helpers/session_timeout_warning_helper.rb

```diff
@@ -18,35 +18,6 @@ def timeout_refresh_path
     )&.html_safe # rubocop:disable Rails/OutputSafety
   end
 
-  def auto_session_timeout_js
-    nonced_javascript_tag do
-      render partial: 'session_timeout/ping',
-             formats: [:js],
-             locals: {
-               timeout_url: timeout_url,
-               warning: warning,
-               start: start,
-               frequency: frequency,
-               modal: modal,
-             }
-    end
-  end
-
-  # rubocop:disable Rails/HelperInstanceVariable
-  def auto_session_expired_js
-    return if @skip_session_expiration
-
-    session_timeout_in = Devise.timeout_in
-    nonced_javascript_tag do
-      render(
-        partial: 'session_timeout/expire_session',
-        formats: [:js],
-        locals: { session_timeout_in: session_timeout_in },
-      )
-    end
-  end
-  # rubocop:enable Rails/HelperInstanceVariable
-
   def time_left_in_session
     distance_of_time_in_words(
       warning,
```

Comment:
looks like this needs to be updated with the rename

Reply:
I think it's fine because it's an explicit local value: https://github.com/18F/identity-idp/pull/4674/files#diff-3330ae0e4620e226f2d9196d9307e13e6bc8a39c3528de7dcc0583d5adfd4ca9R81

Reply:
Big whoops, fixed in 00c2faf9504541281b6c5da4f87d25c39f3254d7